.env

```
@@ -0,0 +1,33 @@
+# 서버
+SERVER_PORT=8080
+BASE_URL=http://localhost:8080
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The BASE_URL key should go before the SERVER_PORT key

(UnorderedKey)